sdk/compute/Azure.ResourceManager.Compute/CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 1.0.0-beta.5 (2021-12-24)
 
 ### Features Added
 
+- Added `CreateResourceIdentifier` for each resource class
+
 ### Breaking Changes
 
+- Renamed `CheckIfExists` to `Exists` for each resource class
+- Renamed `Get{Resource}ByName` to `Get{Resource}AsGenericResources` in SubscriptionExtension
```

Comment:
Please add some new items here:
- Constructor of `OSFamilyCollection`, `OSVersionCollection` no longer accept `location` as their first parameter.
- Constructor of `VirtualMachineExtensionImageCollection` no longer accepts `location` and `publisher` as its first two parameters.
- Method `GetOSFamilies` and `GetOSVersions` in `SubscriptionExtensions` now accept an extra parameter `location`.
- Method `GetVirtualMachineExtensionImages` in `SubscriptionExtensions` now accepts two extra parameters `location` and `publisher`.
- Class `OSFamilyCollection`, `OSVersionCollection` and `VirtualMachineExtensionImageCollection` now implement the `IEnumerable<T>` and `IAsyncEnumerable<T>`

Reply:
Added, but VirtualMachineExtensionImageCollection only implements [IEnumerable<T>](https://github.com/Azure/azure-sdk-for-net/blob/71ec3c1dfdfaa8f536af7ab2f4ca64e2deb18863/sdk/compute/Azure.ResourceManager.Compute/src/Generated/VirtualMachineExtensionImageCollection.cs?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L23).